packages/host/app/components/pill.gts

```diff
@@ -5,85 +5,105 @@ import Component from '@glimmer/component';
 
 import {
   IconButton,
-  Pill,
+  Pill as BoxelPill,
   RealmIcon,
   Switch,
 } from '@cardstack/boxel-ui/components';
-import { cn } from '@cardstack/boxel-ui/helpers';
-import { IconX } from '@cardstack/boxel-ui/icons';
+import { cn, cssVar } from '@cardstack/boxel-ui/helpers';
+import { IconX, CodeFile } from '@cardstack/boxel-ui/icons';
```

Comment:
Don't we already have an icon for this? It's named a bit differently: `import FileCode from '@cardstack/boxel-icons/file-code';`